{"uuid": "29a85a28-7c83-4c17-aa42-363e98046ae0", "children": ["3f1ef14c-3b6d-4f41-aa5c-9648c1aa7e6d", "6ac99b0b-22b6-478d-81da-449d4d76d5b1"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 300.000\n  (Session info: chrome=120.0.6099.225)\nStacktrace:\n\tGetHandleVerifier [0x00007FF6DDAB2142+3514994]\n\t(No symbol) [0x00007FF6DD6D0CE2]\n\t(No symbol) [0x00007FF6DD5776AA]\n\t(No symbol) [0x00007FF6DD560839]\n\t(No symbol) [0x00007FF6DD5606EB]\n\t(No symbol) [0x00007FF6DD55EE3D]\n\t(No symbol) [0x00007FF6DD55F7CE]\n\t(No symbol) [0x00007FF6DD56D6CD]\n\t(No symbol) [0x00007FF6DD584471]\n\t(No symbol) [0x00007FF6DD58937A]\n\t(No symbol) [0x00007FF6DD55FEC6]\n\t(No symbol) [0x00007FF6DD5841CF]\n\t(No symbol) [0x00007FF6DD602824]\n\t(No symbol) [0x00007FF6DD5E5D93]\n\t(No symbol) [0x00007FF6DD5B4BDC]\n\t(No symbol) [0x00007FF6DD5B5C64]\n\tGetHandleVerifier [0x00007FF6DDADE16B+3695259]\n\tGetHandleVerifier [0x00007FF6DDB36737+4057191]\n\tGetHandleVerifier [0x00007FF6DDB2E4E3+4023827]\n\tGetHandleVerifier [0x00007FF6DD8004F9+689705]\n\t(No symbol) [0x00007FF6DD6DC048]\n\t(No symbol) [0x00007FF6DD6D8044]\n\t(No symbol) [0x00007FF6DD6D81C9]\n\t(No symbol) [0x00007FF6DD6C88C4]\n\tBaseThreadInitThunk [0x00007FFCD8FE7344+20]\n\tRtlUserThreadStart [0x00007FFCDA3226B1+33]\n\n", "trace": "  File \"C:\\Users\\Dell\\PycharmProjects\\pythonProject\\pyWebAutomation1x\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\PycharmProjects\\pythonProject\\pyWebAutomation1x\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\PycharmProjects\\pythonProject\\pyWebAutomation1x\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\PycharmProjects\\pythonProject\\pyWebAutomation1x\\tests\\vwoLoginTests\\conftest.py\", line 18, in setup\n    driver.get(base_url)\n  File \"C:\\Users\\Dell\\PycharmProjects\\pythonProject\\pyWebAutomation1x\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 357, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"C:\\Users\\Dell\\PycharmProjects\\pythonProject\\pyWebAutomation1x\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 348, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Dell\\PycharmProjects\\pythonProject\\pyWebAutomation1x\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1705734557102, "stop": 1705734907234}], "start": 1705734557102, "stop": 1705734914693}